odoo/addons/base/models/ir_actions.py

```diff
@@ -556,11 +572,115 @@ def _compute_available_model_ids(self):
+        for field_name in path.split('.'):
+            is_last_field = field_name == path.split('.')[-1]
+            field = model._fields[field_name]
+            if field.relational and not is_last_field:
+                model = self.env[field.comodel_name]
+            elif not field.relational:
+                # sanity check: this should be the last field in the path
+                if not is_last_field:
+                    raise ValidationError(_("The path to the field to update contains a non-relational field (%s) that is not the last field in the path. You can't traverse non-relational fields (even in the quantum realm). Make sure only the last field in the path is non-relational.", field_name))
+                if field.readonly:
+                    raise ValidationError(_("The field to update (%s) is read-only. You can't update a read-only field - even when asking nicely.", field_name))
+        target_records = None
+        if record is not None:
+            target_records = record
+            for field_name in path.split('.'):
+                is_last_field = field_name == path.split('.')[-1]
+                if is_last_field:
+                    # we're at the end of the path, the current field is the one that needs to be updated
+                    break
+                target_records = target_records[field_name]
+        model_id = self.env['ir.model']._get(model._name)
+        field_id = self.env['ir.model.fields']._get(model._name, field_name)
+        return model_id, field_id, target_records
+
+    def _stringify_path(self):
+        """ Returns a string representation of the update_path, with the field names
+        separated by the `>` symbol."""
+        self.ensure_one()
+        path = self.update_path
+        if not path:
+            return ''
+        model = self.env[self.model_id.model]
+        pretty_path = []
+        for field_name in path.split('.'):
+            field = model._fields[field_name]
+            field_id = self.env['ir.model.fields']._get(model._name, field_name)
+            if field.relational:
+                model = self.env[field.comodel_name]
+            pretty_path.append(field_id.field_description)
+        return ' > '.join(pretty_path)
+
+    @api.depends('state', 'webhook_field_ids', 'name')
+    def _compute_webhook_sample_payload(self):
+        for action in self:
+            if action.state != 'webhook':
+                action.webhook_sample_payload = False
+                continue
+            payload = {
+                '_id': 1,
+                '_model': self.model_id.model,
+                '_name': action.name,
+            }
+            for field in action.webhook_field_ids:
+                sample_record = self.env[self.model_id.model].with_context(active_test=False).search([], limit=1, order='active desc, id desc')
```

Comment:
what if there is no active field on the model, order would fail
remove the order arg

Reply:
move outside of the for loop 🤦